packages/core/test/Serialization/Scripts.test.ts

```diff
@@ -0,0 +1,615 @@
+/* eslint-disable sonarjs/no-duplicate-string */
+import * as Crypto from '@cardano-sdk/crypto';
+
+import { Cardano, Serialization } from '../../src';
+import { HexBlob } from '@cardano-sdk/util';
+
+describe('Scripts', () => {
+  it('can decode the nested native script from CBOR', () => {
+    const script = Serialization.Script.fromCbor(
+      HexBlob(
+        '82008202828200581cb275b08c999097247f7c17e77007c7010cd19f20cc086ad99d3985388201838205190bb88200581c966e394a544f242081e41d1965137b1bb412ac230d40ed5407821c378204190fa0'
```

Comment:
Would be great to put in some inline comments about the source (or how test data was produced) for all CBORs and hashes